### ISSUE TITLE
Editing a device in a cluster throws exceptions.ValueError

### ISSUE THREAD

**Issue by HumanEquivalentUnit** [digitalocean/netbox#2611]
Environment: Netbox 2.4.8 on Python 2.7 on CentOS

Steps:
- Find a device (server) in a virtualization cluster, and edit it on the Device screen
- change its location, (site, rack, position)
- click the "update button"

Expected: location gets updated.

Observed: update fails with this error:

```
<type 'exceptions.ValueError'>

'DeviceForm' has no field named 'cluster'.
```

Workaround is to go to Virtualizations -> Clusters, remove the server from the cluster, then go back and change its location.

**Comment by mkerins** [comment]
I'm not able to reproduce this issue, please take a look at this [gist](https://gist.github.com/mkerins/71da0252fac12db1e45ade9c93bcfa69) and compare to yours. I was able to update the position of the device with no error. Thanks.

**Comment by HumanEquivalentUnit** [comment]
@mkerins I can reproduce it by making a test cluster again, adding a device back into it, and trying to edit its location. I see this issue has been closed and addressed so it may be not worth pursuing, but just curious - what is the json dump of content shown in your gist? How can I get that out of my Netbox so I can compare, or import it in to a test one?

(I hope this doesn't reopen the issue, sorry if it does).

**Comment by mkerins** [comment]
@HumanEquivalentUnit sorry I wasn't clear in my reply. The JSON is the output of the database showing what I entered in to try and replicate your issue. It looks like it's been resolved with v2.4.9 anyways, please try upgrading and see if it still occurs.
